sandbox/src/mocks/retrieveWorklist/requests/MinimalAwaitingBooking.json

```diff
@@ -0,0 +1,22 @@
+{
+    "resourceType": "Parameters",
+    "meta": {
+      "profile": [
+        "https://fhir.nhs.uk/STU3/StructureDefinition/eRS-FetchWorklist-Parameters-1"
+      ]
+    },
+    "parameter": [
+      {
+        "name": "listType",
+        "valueCodeableConcept": {
+          "coding": [
+            {
+              "system": "https://fhir.nhs.uk/STU3/CodeSystem/eRS-ReferralListSelector-1",
+              "code": "AWAITING_BOOKING"
```

Comment:
We'll need to make sure that these new values to be added to the ReferralListSelector code system are passed over to IOPS so they can be added to the static code system https://fhir.nhs.uk/STU3/CodeSystem/eRS-ReferralListSelector-1

Reply:
Do I need to contact someone from IOPS to make sure this happens or is there another process we follow for this?

Reply:
Do I need to contact someone from IOPS to make sure this happens or is there another process we follow for this?

Reply:
Covered by https://nhsd-confluence.digital.nhs.uk/pages/viewpage.action?pageId=384402460